ui/core/src/actions/ethWallet.test.ts

```diff
@@ -47,6 +57,7 @@ beforeEach(() => {
 test("Calls disconnect", async () => {
   await ethWalletActions.disconnectWallet();
   expect(mockEthereumService.disconnect).toHaveBeenCalled();
+  expect(notify).toHaveBeenCalled();
```

Comment:
I just want to point out that by having notify in a service it is easier to test that the notification happened (something we will be more and more concerned with moving forward)